builds/zika/Snakefile

```diff
@@ -9,118 +9,119 @@ rule all:
 # the advantage of this is that one can (eventually) import rules from a template
 # currently only files are defined here - parameters are defined in their own rules
 rule config:
-	params:
-		input_fasta = "../../../fauna/data/zika.fasta",
-		fasta_fields = "strain virus accession date region country division city db segment authors url title journal paper_url",
-		dropped_strains = "config/dropped_strains.txt",
-		reference = "config/zika_outgroup.gb",
-		geo_info = "../../source-data/geo_lat_long.tsv",
-		color_maps = "config/colors.tsv",
-		auspice_config = "config/auspice_config.json"
+    params:
+        input_fasta = "../../../fauna/data/zika.fasta",
+        fasta_fields = "strain virus accession date region country division city db segment authors url title journal paper_url",
+        dropped_strains = "config/dropped_strains.txt",
+        reference = "config/zika_outgroup.gb",
+        geo_info = "../../source-data/geo_lat_long.tsv",
+        color_maps = "config/colors.tsv",
+        auspice_config = "config/auspice_config.json"
 
 config = rules.config.params # so we can use config.x rather than rules.config.params.x
 # end of config definition
 
 rule parse:
-	input:
-		config.input_fasta
-	output:
-		sequences = "results/sequences.fasta",
-		metadata = "results/metadata.tsv"
-	shell:
-		'augur parse --sequences {input} --output {output.sequences} --metadata {output.metadata} --fields {config.fasta_fields}'
+    input:
+       config.input_fasta
+    output:
+       sequences = "results/sequences.fasta",
+       metadata = "results/metadata.tsv"
+    shell:
+        "augur parse --sequences {input} --output {output.sequences} --metadata {output.metadata} "
+            "--fields {config.fasta_fields}"
 
 
 rule filter:
-	input:
-		sequences = rules.parse.output.sequences,
-		metadata = rules.parse.output.metadata,
-		exclude = config.dropped_strains
-	output:
-		"results/filtered.fasta"
-	params:
-		vpc = 1,
-		cat = "year month",
-		min_date = 2012
-	shell:
-		"augur filter --sequences {input.sequences} --output {output} --metadata {input.metadata} "
-			"--viruses-per-cat {params.vpc} "
-			"--exclude {input.exclude} --cat {params.cat} --min-date {params.min_date}"
+    input:
+        sequences = rules.parse.output.sequences,
+        metadata = rules.parse.output.metadata,
+        exclude = config.dropped_strains
+    output:
+        "results/filtered.fasta"
+    params:
+        vpc = 20,
```

Comment:
@trvrb This otherwise whitespace-only commit also changed zika's `vpc` parameter from 1 to 20. Was that intended?

Reply:
Yes. It should have been 20 all along. See here: https://github.com/nextstrain/augur/blob/master/builds/zika/zika.prepare.py#L16. The "1" was the mistake.